apps/site-nft/src/i18n.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable global-require */
+/**
+ * i18n.js
+ *
+ * This will setup the i18n language files and locale data for your app.
+ */
+if (!Intl.PluralRules) {
+  require('@formatjs/intl-pluralrules/polyfill');
+  require('@formatjs/intl-pluralrules/dist/locale-data/en'); // Add locale data for en
+  require('@formatjs/intl-pluralrules/dist/locale-data/fr'); // Add locale data for fr
+}
+
+interface Messages {
+  [index:string]: string;
+}
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const enTranslationMessages: Messages = require('./translations/en.json');
+const frTranslationMessages: Messages = require('./translations/fr.json');
```

Comment:
This i18n file is significantly simpler than the original.  I'm not entirely sure, but I think most of what the original code did is no longer necessary